docs/guides/modules/deploy/pages/set-up-rollbacks.adoc

```diff
@@ -35,97 +35,88 @@ a| * Simple systems
-
-You will see a red Rollback button with a dropdown option on the project overview page.
+=== Start the rollback pipeline setup
 
-.Rollback options on project overview page
+. In the CircleCI dashboard, navigate to *Organization Home* from the sidebar.
+. Select the menu:Overview[] link for your project.
+. In the menu that appears, select btn:[Create rollback pipeline].
++
 image::guides:ROOT:deploy/project-overview-rollback.png[Rollback button on project overview page]
 
-The following steps guide you through setting up a custom rollback pipeline. If you want details on how to *Rollback by workflow re-run*, see the <<rollback-by-workflow-re-run, Rollback by workflow re-run>> section at the end of this guide.
+=== Confirm GitHub App installation
 
-=== 2. Start the rollback setup
+NOTE: To set up rollback pipelines, you must connect your organization to the xref:integration:github-apps-integration[CircleCI GitHub App].
 
-. Select the btn:[Rollback] dropdown.
-. Select *Set up custom rollback pipeline* from the dropdown menu. This launches the rollback setup wizard.
+- If the GitHub App is not installed in your organization, the guided setup process prompts you to install it.
 +
-NOTE: To use the rollback by pipeline method, your organization must be connected to the *CircleCI GitHub App*. If the GitHub App is not installed in your org, the rollback setup process will automatically prompt you to install it during setup.
-
-The setup wizard guides you through configuring your custom rollback pipeline.
-
-image::guides:ROOT:deploy/set-up-a-rollback-pipeline.png[Set up a rollback pipeline]
+image::guides:ROOT:deploy/install-github-app.png[Install GitHub App]
 
-=== 3. Configure the rollback pipeline
-
-. From the "What repo are you deploying?" dropdown, select the repository you want to create rollbacks for.
-. Select btn:[Create pipeline definition] to proceed.
+- If the GitHub App is already installed in your organization, the **GitHub App Installation** step is automatically marked as completed.
++
+image::guides:ROOT:deploy/github-app-installed.png[GitHub App installed]
 
-CircleCI creates a pipeline definition called `rollback-pipeline` and uses the selected repository to store your rollback configuration.
 
-=== 4. Review the generated configuration
-After creating the pipeline definition, the modal will display a pre-generated configuration file for performing rollbacks.
+=== Add the deploy markers
 
-. Review the generated YAML configuration template to ensure you understand the rollback pipeline.
-. Select btn:[Commit Config] to create this rollback configuration template in your repository. The configuration will be committed to a new branch called `rollback-pipeline-setup` in your selected repository.
+. Expand the **Add deploy markers to existing pipeline** section.
+. In the **Environment Name** dropdown, select or type the xref:configure-deploy-markers#manage-environments[target environment].
```

Comment:
Add something here about what environments are - can you the same language as the tooltip in the app actually. Also mention if you don't have any set up yet adding one here will set up an environment of that name for you when a pipeline runs .... I think that's correct anyway

(Same feedback for numxt bullet point too)